### PR TITLE
Allow useCode 1st parameter to return undefined, null or an array

### PR DESCRIPTION
**Pull request by yairopro**
And makes the code clearer (it seems clearer to me 😜)

### CODE REVIEW

**Comment by osdnk** [comment]
Shouldn't you fix typings or docs?

**Comment by yairopro** [comment]
What do you mean ?

**Comment by osdnk** [comment]
Are you that you don't need to change typescript definitions and describe your changes in docs?

File: src/derived/useCode.js [hunk 1]
```diff
@@ -1,20 +1,41 @@
 import React from 'react';
-import { always } from '../base';
+import {always, block} from '../base';
```
Comment (SaeedZhiany) [inline src/derived/useCode.js]: `import { always, block } from '../base';`

put space after and before brackets, it's a code style in ES6 importing

**Comment by yairopro** [comment]
> Are you that you don't need to change typescript definitions and describe your changes in docs?

Oh sorry ! I'm not used to TypeScript, but of course you're right, I'm going to update definitions right away. Thanks fo telling me. 
So maybe the documentation won't be needed anymore.

File: src/derived/useCode.js (hunk 1 above)
Comment (yairopro) [reply src/derived/useCode.js]: OK I will edit
```suggestion
import {always, block} from '../base';
```

**Comment by yairopro** [comment]
@osdnk TypeScript definition updated.
Be my guess and have a look please.😁

**Comment by osdnk** [comment]
It should be mentioned in docs :)

**Comment by yairopro** [comment]
@osdnk 
JSDoc updated & TypeScript definitions corrected (`nodeFactory` function can returns an array of nodes).

**Comment by osdnk** [comment]
Sorry, I didn't want to close.
Please mention your changes :)

**Comment by osdnk** [comment]
Sorry, i wanted to say: Please mention your changes in documentation

**Comment by yairopro** [comment]
@osdnk 
I think it's ready.

**Comment by yairopro** [comment]
😁 Thanks @osdnk

**Pull request merged**